src/main/java/seedu/us/among/logic/commands/AddCommand.java

```diff
@@ -17,15 +17,10 @@
     public static final String COMMAND_WORD = "add";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds an API endpoint to the API endpoint list. "
-            + "Parameters: "
-            + PREFIX_NAME + "NAME "
-            + PREFIX_ADDRESS + "ADDRESS "
-            + "[" + PREFIX_TAG + "TAG]...\n"
-            + "Example: " + COMMAND_WORD + " "
-            + PREFIX_NAME + "John Doe "
-            + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
-            + PREFIX_TAG + "friends "
-            + PREFIX_TAG + "owesMoney";
+            + "Parameters: " + PREFIX_METHOD + "METHOD " + PREFIX_ADDRESS + "ADDRESS " + "[" + PREFIX_TAG + "TAG]...\n"
```

Comment:
Will be good to follow the spacing format shown in other MESSAGE_USAGE classes for more readability.

Reply:
![image](https://user-images.githubusercontent.com/19247856/109930251-4a57f780-7c7c-11eb-826a-b142a3ca4c9a.png)
Don't worry, I do not allow duplicate endpoints, the check is as follows -> if the method and address same it is considered as a duplicate.

Reply:
Also wdym by this comment, on my side, it looks the same as the other commands.